eventstream.py

```diff
@@ -20,11 +20,14 @@
 import sys
 
 if sys.version[0] == '2':
```

Comment:
If we're not going to support python 2, I'd prefer all the related code just be removed rather than commented out.